cluster_agent/jobbergate/schemas.py

```diff
@@ -3,16 +3,23 @@
 
 import pydantic
 
-from cluster_agent.jobbergate.constants import JobSubmissionStatus, status_map
+from cluster_agent.jobbergate.constants import FileType, JobSubmissionStatus, status_map
 
 
-class JobScriptFiles(pydantic.BaseModel, extra=pydantic.Extra.ignore):
-    """
-    Model containing job-script files.
-    """
+class JobScriptFile(pydantic.BaseModel, extra=pydantic.Extra.ignore):
+    """Model for the job_script_files field of the JobScript resource."""
+
+    id: int
+    filename: str
+    file_type: FileType
 
-    main_file_path: Path
-    files: Dict[Path, str]
+    url: str
```

Comment:
Rename to `path` to be consistent with the schemas from the API and CLI

Reply:
solved in [8077a86](https://github.com/omnivector-solutions/cluster-agent/pull/71/commits/8077a860e89b058684a687f1694fe2e4887bfaf8)